packages/ui/src/components/va-pagination/VaPagination.demo.vue

```diff
@@ -1,216 +1,376 @@
 <template>
-  <VbDemo>
+  <VbDemo class="va-pagination-demo">
     <VbCard>
       Current Page
       <input
-        type="number"
         v-model.number="activePage"
+        aria-hidden="true"
+        type="number"
       />
     </VbCard>
     <VbCard>
       <table class="table table-bordered">
         <tr>
           <th>Description</th>
-          <th>Va Pagination</th>
+          <th>VaPagination</th>
```

Comment:
<img width="241" alt="image" src="https://user-images.githubusercontent.com/5394573/189230603-3c13d1f1-8221-444c-8c38-000c47edb23e.png">

When you click on page - you get this orange border. Very annoying :).

Reply:
It's already solved. Just not in this issue branch. Will merge 1.5.0 to it and everything will be nice :)